src/scripts/helpers/zotonic_setup

```diff
@@ -40,8 +40,7 @@ export -f require_zotonic_not_running
 function enablesite {
     config="priv/sites/$1/config"
     expr="s/enabled, \(true\|false\)/enabled, $2/"
-    perl -e "$expr" -pi $config
-    #echo sed -e "$expr" -i $config
+    sed -e "$expr" -i.bak -e $config
```

Comment:
I see that on [BSD](https://www.freebsd.org/cgi/man.cgi?query=sed) it's supported.